src/components/utils/input.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <label class="cc-input">
+    <div v-if="!hideLabel" class="cc-label">
+      {{label}}
+    </div>
+
+    <div class="cc-input__body" :class="{'focused': isFocused}">
+      <input
+        class="cc-input__input"
+        v-model="validation"
+        @input="$emit('input, $event.target.value')"
```

Comment:
should be :value instead of v-model binding => we shouldn't mutate "value" prop directly
move input event to bottom
fix event syntax (move ' to "input" word)